### DISCUSSION TITLE
Not possible to send direct messages

### DISCUSSION THREAD

**Discussion by olemara** [mastodon/mastodon#27244] [Q&A]
Hi,

I have created the following issue regarding this: https://github.com/mastodon/mastodon/issues/27232

The problem is very strange and appeared a couple of weeks ago. I am still struggling to debug this in a sensible way. I have tried installing 4.0.7 and 4.0.11 as well without luck.

Any input on how to proceed in debugging this?

**Comment by olemara** [comment]
Same thing happening with v4.0.7, v4.0.11, and v4.0.1.

Clearing all client side data when testing of course.

**Comment by Shanesan** [comment]
There's no way for this community to help if you don't tell us the steps taken to produce the bug.

**Comment by olemara** [comment]
Hi,

@Shanesan, sorry for being unclear, there was not any specific steps except using the functionality on specified version(s). In #27243 the issue has been resolved as there was some discrepancy between versions that worked before. This is due to updates made to the images the last weeks.
